Apps/W1/ReviewGLEntries/app/src/tables/GLEntryReviewLog.Table.al

```diff
@@ -0,0 +1,46 @@
+namespace Microsoft.Finance.GeneralLedger.Review;
```

Comment:
In General I'd reccomend to add DrillDownPageId here.
We might want to disable the DrillDown then for some fields on the GLEntries Pageextension.